### DISCUSSION TITLE
Can't run this library on android 10+

### DISCUSSION THREAD

**Discussion by cakhsoft** [schwabe/ics-openvpn#1545] [General]
I have run this code on android 10+ and it is not working on those device versions. Does anybody have a solution for that?

**Comment by schwabe** [comment]
The app works perfectly fine on Android 10.

**Reply by cakhsoft** [reply]
When building app from bundles, the process terminates during runtime with:
CANNOT LINK EXECUTABLE "/data/user/0/de.blinkt.openvpn/cache/c_pie_openvpn.x86": library "libopenvpn.so" not found

**Reply by schwabe** [reply]
Try to build the app without modifications and see if that works for you.

**Reply by cakhsoft** [reply]
ok let me try brother
